packages/features/ee/workflows/components/WorkflowDetailsPage.tsx

```diff
@@ -71,7 +78,8 @@ export default function WorkflowDetailsPage(props: Props) {
       emailSubject: null,
       template: WorkflowTemplates.CUSTOM,
       numberRequired: numberRequired || false,
-      sender: sender || SENDER_ID,
+      sender: isSMSAction(action) ? sender || SENDER_ID : SENDER_ID,
```

Comment:
This may be more readable as an if statement later on as there's a toggling ternary here on these two lines.